app/views/compute_resources/show.html.erb

```diff
@@ -17,11 +17,11 @@
 
 <div class="tab-content">
   <div class="tab-pane active" id="primary">
-    <table class="table table-bordered table-striped">
+    <table class="table table-bordered table-striped table-fixed">
       <thead>
         <tr>
-          <th><%= _("Details") %></th>
-          <th><%= _("Actions") %></th>
+          <th class="col-md-1"><%= _("Details") %></th>
+          <th class="col-md-6"><%= _("Actions") %></th>
```

Comment:
Maybe "Property" - "Value"?  Doesn't really matter to me as long as it's clear. Make your choice.